### DISCUSSION TITLE
Add additional SANs

### DISCUSSION THREAD

**Discussion by TonyJHART** [Venafi/venafips#107] [Q&A] [answered]
I am not realizing how to add a SAN to a  cert.  I know how for a  totally new cert.  
Any pointers? Or is API calls the only way.

**Comment by gdbarron** [comment, marked as answer]
@TonyJHART, you can use `Set-TppAttribute` to update certificate values.  For example, to add a SAN DNS you would use `Set-TppAttribute -Path '\ved\policy\mycert.com' -Attribute @{'X509 SubjectAltName DNS'='site.domain.com'}`.

To get the list of available attributes for a specific class, use `Get-TppClassAttribute`.  For example, `Get-TppClassAttribute -ClassName 'X509 Server Certificate'`.

As an aside, if you ever did need to call an api that didn't have a dedicated function, you can use `Invoke-VenafiRestMethod`.
